packages/cloud_firestore/cloud_firestore/android/src/main/java/io/flutter/plugins/firebase/firestore/FlutterFirebaseFirestoreMessageCodec.java

```diff
@@ -301,6 +313,8 @@ private FirebaseFirestore readFirestoreInstance(ByteBuffer buffer) {
       FirebaseFirestore firestore = FirebaseFirestore.getInstance(app, databaseURL);
       firestore.setFirestoreSettings(settings);
 
+      firestore.setLoggingEnabled(false);
```

Comment:
Why is this set to false?

Reply:
It defaults to false. I just never remember where to turn it on, so thought. I would set it. This turns on firebase_android_sdk logging which is alot of logging.